benchmark.cc

```diff
@@ -82,20 +92,23 @@ graphStat testBuildTime(const uint64_t mapSize, MAP_TYPE mapType, double loadFac
             std::cerr << "Unexpected map type\n";
             exit(1);
     }
-    graphStat graphStatistics(mapType, loadFactor, mapSize);
+    graphStat graphStatistics(mapType, maxLoadFactor, mapSize);
 
-    if(verbose >= 1) {
-        std::cout << "Initializing " << graphStatistics.getType() << " with a load factor of " << loadFactor << "\n";
+    if(verbose_ >= 1) {
+        std::cout << "Initializing " << graphStatistics.getType() << " with a max load factor of " << maxLoadFactor << "\n";
     }
+    auto time1 = std::chrono::high_resolution_clock::now();
+
     //populate map
     for(uint64_t i = 0; i < mapSize; ++i){
+        //this overflows around i = 3 billion, but that shouldn't cause any problems as the value will just wrap back to 0
         map->put(i, (i+1)*(i+3));
     }
 
-    time2 = std::chrono::high_resolution_clock::now();
+    auto time2 = std::chrono::high_resolution_clock::now();
```

Comment:
Nice.
I like the `chrono` library.  Code that uses it is so much more readable.